docs/source/install.rst

```diff
@@ -63,10 +64,21 @@ packages are specific to different operating systems:
 * macOS (OS X): `MacTeX <http://tug.org/mactex/>`_.
 * Windows: `MikTex <http://www.miktex.org/>`_
 
-Because nbconvert depends on packages and fonts included in standard 
-TeX distributions, if you do not have a complete installation, you 
-may not be able to use nbconvert's standard tooling to convert 
-notebooks to PDF. 
+Because nbconvert depends on packages and fonts included in standard
+TeX distributions, if you do not have a complete installation, you
+may not be able to use nbconvert's standard tooling to convert
+notebooks to PDF.
+
+Installing Chromium
+-------------------
+
+When converting to PDF with ``--to webpdf``, nbconvert uses `Puppeteer <https://pptr.dev>`_, and
+more precisely its Python bindings `pyppeteer <https://github.com/pyppeteer/pyppeteer>`_. pyppeteer
```

Comment:
Again, puppeteer is not used. Since there's less going on, perhaps moving this section above the TeX section makes sense.